test/Microsoft.Extensions.Configuration.Binder.Test/Microsoft.Extensions.Configuration.Binder.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;net46</TargetFrameworks>
```

Comment:
got it. I was assuming they were synonyms but it seems things are more complicated than I thought :smile: